src/init/features/apphosting/index.ts

```diff
@@ -132,6 +144,96 @@ export async function doSetup(
   logSuccess(`Your site is now deployed at:\n\thttps://${backend.uri}`);
 }
 
+/**
+ *
+ * @param projectId user's projectId
+ * @param webAppName (optional) name of web app
+ * @return app name and app id
+ */
+async function getOrCreateWebAppId(
+  projectId: string,
+  webAppName: string | null,
+  backendId: string,
+): Promise<string[]> {
+  let webAppId: string;
+
+  const existingUserProjectWebApps = new Map(
+    (await listFirebaseApps(projectId, AppPlatform.WEB)).map((obj) => [
+      // displayName can be null, use name instead if so. Example - displayName: "mathusan-web-app", name: "projects/mathusan-fwp/webApps/1:461896338144:web:426291191cccce65fede85"
```

Comment:
I'm sure this will be addressed in the API review but it may be better to print out just the actual name of the web app, ie `1:461896338144:web:426291191cccce65fede85`

Reply:
do you mean we show the `appId` instead of the `appName` when `displayName` isn't available?

Reply:
I actually meant `1:461896338144:web:426291191cccce65fede85` instead of `projects/mathusan-fwp/webApps/1:461896338144:web:426291191cccce65fede85`